storage/innobase/xtrabackup/src/fil_cur.cc

```diff
@@ -242,6 +242,19 @@ xb_fil_cur_open(
 
 	/* Allocate read buffer */
 	cursor->buf_size = XB_FIL_CUR_PAGES * page_size;
+	if (opt_read_buffer_size != 0) {
+		ulint buf_size = cursor->buf_size;
+		const char *factors[] = {"b", "Kb", "Mb", NULL};
```

Comment:
This buffer is for innodb files only. What about other ones? MyISAM files in particular can actually be quite large.